Task2/Task2/Task2/Products/Bolt.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Task2.Exceptions;
+
+namespace Task2.Products
+{
+    public class Bolt : Product
+    {
+        public Bolt(string productName, double purchaseCost, double margin, int numberOfUnits)
+            : base(productName, purchaseCost, margin, numberOfUnits)
+        {
+
+        }
+
+        public static Bolt operator +(Bolt leftProduct, Bolt rightProduct)
+        {
+            if (leftProduct.ProductName.Equals(rightProduct.ProductName, StringComparison.InvariantCultureIgnoreCase))
+            {
+                var obj = new Bolt("", 0, 0, 0)
```

Comment:
Just create an empty private constructor and delete all in ( ).

Reply:
fixed